compeon-rack_tools.gemspec

```diff
@@ -15,6 +15,8 @@ Gem::Specification.new do |spec|
   spec.homepage      = 'https://github.com/COMPEON/compeon-rack_tools'
   spec.license       = 'MIT'
 
+  spec.metadata['allowed_push_host'] = 'https://rubygems.pkg.github.com/COMPEON'
```

Comment:
Without looking into the specification, I would expect `...host` to expect a host rather than a full URL. Will look that up though.

Reply:
https://guides.rubygems.org/publishing/#serving-your-own-gems

Reply:
This does not really clarify my doubts as the documentation only passes in protocol and a TLD.

Reply:
Maybe we should just try it. 🤷‍♂️
All examples/docs that I found so far did not specify a path.

Reply:
FTR: https://github.com/rubygems/rubygems/issues/3093